lib/search/query_components/core_query.rb

```diff
@@ -62,17 +71,15 @@ def quoted_phrase_query
     end
 
     def unquoted_phrase_query
-      [
-        dis_max_query([
-          match_phrase("title", PHRASE_MATCH_TITLE_BOOST),
-          match_phrase("acronym", PHRASE_MATCH_ACRONYM_BOOST),
-          match_phrase("description", PHRASE_MATCH_DESCRIPTION_BOOST),
-          match_phrase("indexable_content", PHRASE_MATCH_INDEXABLE_CONTENT_BOOST),
-          match_all_terms(%w(title acronym description indexable_content)),
-          match_any_terms(%w(title acronym description indexable_content), 0.2),
-          minimum_should_match("all_searchable_text", 0.2)
-        ], tie_breaker: 0.7)
-      ]
+      should_coord_query([
```

Comment:
Unquoted search queries on the ES6 cluster get sent here, but I think we're using the same quoted search query for both ES5 and 6.  Is that right?  Is that OK?

Reply:
Yes, I haven't changed the quoted search query.  That does exact matching, as it should, so I'm not sure there's anything to change there.  We might want to combine the two types of query in the future, but that's a separate card.

Reply:
Cool, I'm happy then 👍